src/lib/lwan.c

```diff
@@ -543,6 +582,24 @@ void lwan_init_with_config(struct lwan *l, const struct lwan_config *config)
 
     try_setup_from_config(l, config);
 
+    l->headers = lwan_strbuf_new();
+    bool set_server = false;
+    if (UNLIKELY(!!l->config.global_headers)) {
+        for (struct lwan_key_value *kv = l->config.global_headers; kv->key; ++kv) {
+            set_server = !strcasecmp(kv->key, "server");
```

Comment:
Please use something like the following, as the code as written won't work unless `Server`  is the last key in the array.

```suggestion
            if (!strcasecmp(kv->key, "server"))
            	set_server = true;
```